client/ting.jsx

```diff
@@ -92,7 +92,8 @@ const Ting = React.createClass({
             type: 'channel',
             target: this.state.channel,
             text: message,
-            messageid: this.state.currentMessageId
+            messageid: this.state.currentMessageId,
+            message_type: messageType
```

Comment:
This is an API inconsistency; this is the only key with an underscore and a prefix. If we change the name appropriately, it will look better without a prefix.